opensearch-project/OpenSearch-Dashboards
# [Workspace]Disable "Save changes" button on character limit exceeded

### Description

This PR is for disable the 'Save Changes' button when the number of characters in the name or description exceeds the maximum limit.

### Issues Resolved



#8154

## Screenshot


<img width="1449" alt="image" src="https://github.com/user-attachments/assets/0a53c9f6-0516-41cf-9fa6-6e2702d499a7">


## Testing the changes



- Clone branch code and run `yarn osd bootstrap --single-version ignore`
- Add below configs in `config/opensearch_dashboards.yml`
```
savedObjects.permission.enabled: true
workspace.enabled: true
uiSettings:
  overrides:
    'home:useNewHomePage': true
opensearchDashboards.dashboardAdmin.users: ['admin']
```
- Run `yarn start --no-base-path`
- Visiting a existing workspace settings page
- Click "Edit" button
- Input a long workspace name or description, the "Save changes" button should be disabled.


## Changelog

- fix: [Workspace]Disable "Save changes" button on character limit exceeded

### Check List
